tests/test_h2_client.c

```diff
@@ -22,6 +23,9 @@
     AWS_TEST_CASE(NAME, s_test_##NAME);                                                                                \
     static int s_test_##NAME(struct aws_allocator *allocator, void *ctx)
 
+#define DEFINE_HEADER(NAME, VALUE)                                                                                     \
```

Comment:
I almost wonder if we should put this in a public header. It's going to be everywhere in our code using this.

Reply:
aws_http_header_from_c_str(const char *name, const char *value, enum aws_http_header_compression compression);

struct aws_http_header r